docs/source/howto/authentication.rst

```diff
@@ -87,8 +87,7 @@ methods:
 3. User account credentials.
 
    pandas-gbq loads cached credentials from a hidden user folder on the
-   operating system. Override the location of the cached user credentials
-   by setting the ``PANDAS_GBQ_CREDENTIALS_FILE`` environment variable.
+   operating system.
```

Comment:
Tangential, but should we say where this is?

Reply:
Done. Added a few more examples regarding user auth, too.